<!-- -->

<launch>
	<node pkg="lidar_camera_fusion" type="fusion.py" name="fusion_node" output="screen">
		<param name ="image_topic" value="/usb_cam/image_rect_color"/>
		<param name ="lidar_topic" value="/pandar_points"/>
		<param name ="calibration_file_path" value="$(find lidar_camera_fusion)/conf/head_camera.yaml"/>
		<param name ="the_sensor_height" value="1.8"/>
		<param name ="the_view_higher_limit" value="2.0"/>
		<param name ="the_view_lower_limit" value="0.0"/>
		<param name ="the_min_distance" value="2.0"/>
		<param name ="the_max_distance" value="50.0"/>

		<param name ="jet_color" value="5"/>

	</node>
</launch>
